test/JDBC/input/declare_atatglobalvars-vu-verify.sql

```diff
@@ -0,0 +1,212 @@
+declare @@cursor_rows int
```

Comment:
declare_atatglobalvars For this file , I think we need to add it to the latest/schedule file as well

Reply:
done